charts/tusd/values.yaml

```diff
@@ -51,6 +51,10 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template.
   name: ""
 
+  # -- Automount API credentials for a service account.
+  # See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#service-account) for details.
+  automountServiceAccountToken: false
```

Comment:
`serviceAccount.automountServiceAccountToken` feels a bit redundant. How about calling it `automountToken` instead?

Also, I understand the security implications, but it feels like most people would expect this to be true by default (and it's also a breaking change).

If there is a wider consensus (or at least discussion) in the community to change the default value, I'd be willing to change it, but I'm not aware that this is happening. (Is it?)

WDYT?

Reply:
I kept the original name to allow people to know what it does, I can shorten it if you like.

About the breaking change, you are correct. Although I would be surprised if any tusd installation needs kubeapi access. Please correct me if I am wrong. I might be missing a feature.

There is indeed no wider consensus on changing this. I just chose the most 'secure' default. Although I am fine changing it. As long as users can change the value :-)